### PR TITLE
[IMP] [12.0] sale_commission - agent lines domain in a specific function

### PR DESCRIPTION
**Pull request by odooNextev**
We need to edit agent lines domain for a specific payment term defined by a module in `l10n-italy` repo and we thought to add a method to return the domain instead including this in the whole function.
In fact, at the moment, we'd overwrite the whole `action_settle` method breaking the chain of inheritance.

### CODE REVIEW

**Comment by odooNextev** [comment]
Hi @pedrobaeza,
can you help us to fix this https://app.travis-ci.com/github/OCA/commission/jobs/598484275#L283?

Btw, have we to add `.pre-commit-config.yaml` to this repo?

**Comment by stenext** [comment]
@pedrobaeza I fixed what you told me for 14.0 module version 
https://github.com/OCA/commission/pull/407#discussion_r1145972294

**Comment by pedrobaeza** [comment]
There are files that shouldn't be modified here, and please put only commit.

**Comment by pedrobaeza** [comment]
And please check CI.

**Comment by stenext** [comment]
> There are files that shouldn't be modified here, and please put only commit.

are you referring to .gitignore?
I should add VS Code section because it wasn't there and I need for my development, but I guess many users need it.
I copied these lines from a .gitignore in another OCA repo

**Comment by pedrobaeza** [comment]
That should be done in another PR, not here.

**Comment by stenext** [comment]
> And please check CI.

I did, but I don't understand if it's a problem in CI test environment or thers's something about our changes
https://app.travis-ci.com/github/OCA/commission/jobs/598721434#L283

We didn't change anything about `pylint-plugin-utils`

**Comment by pedrobaeza** [comment]
That's the problem with such old branches: the CI is now outdated, and you need to perform (in other PR), the transformation to the new CI through `copier` template.

**Comment by odooNextev** [comment]
> That's the problem with such old branches: the CI is now outdated, and you need to perform (in other PR), the transformation to the new CI through `copier` template.

https://github.com/OCA/commission/pull/408

**Comment by tafaRU** [comment]
#408 was merged so please rebase.

File: sale_commission/wizard/wizard_settle.py [hunk 1]
```diff
@@ -123,3 +120,10 @@ def action_settle(self):
 
         else:
             return {'type': 'ir.actions.act_window_close'}
+
+    def _get_agent_lines(self, agent, date_to_agent):
+        aila = self.env['account.invoice.line.agent'].search([
```
Comment (tafaRU) [inline sale_commission/wizard/wizard_settle.py]: Nitpick: could you please avoid this assignment and return directly?

**Review by tafaRU — APPROVED** [review]

File: sale_commission/wizard/wizard_settle.py (hunk 1 above)
Comment (odooNextev) [reply sale_commission/wizard/wizard_settle.py]: I did it

**Pull request merged**